guides/08_gradio-clients-and-lite/07_fastapi-app-with-the-gradio-client.md

```diff
@@ -1,8 +1,8 @@
-# Building a FastAPI App with the Gradio Python Client
+# Building a Web App with the Gradio Python Client
```

Comment:
consider also changing the name of the guide

Reply:
I think its fine to keep the url as it is (I'm also being lazy and don't want to add the redirect)